Auth0/WebAuth.swift

```diff
@@ -118,6 +118,15 @@ public protocol WebAuth: Trackable, Loggable {
     func scope(_ scope: String) -> Self
 
     /**
+      Provider scopes for oauth2/social connections. e.g. Facebook, Google etc
+
+     - parameter connectionScope: oauth2/social scope list: `user_friends email`
```

Comment:
It works with either, I went with spaces as it was consistent with setting scope.